app/models/ability.rb

```diff
@@ -0,0 +1,33 @@
+class Ability
+  include CanCan::Ability
+
+  def initialize(user)
+    user ||= User.new # guest user (not logged in)
+    if user.role == 'admin'
+      can :manage, :all
```

Comment:
[Optional]
- [ ] You can create the `admin?` method in the user model to easily check if the user has a proper role.

```ruby
   def admin?
     role == 'admin' ? true : false
   end
```